packages/web/src/components/gcds-card/gcds-card.tsx

```diff
@@ -0,0 +1,108 @@
+import { Element, Component, Host, Prop, h, Fragment } from '@stencil/core';
+
+@Component({
+  tag: 'gcds-card',
+  styleUrl: 'gcds-card.css',
+  shadow: true,
+})
+export class GcdsCard {
+  @Element() el: HTMLElement;
+
+  /**
+   * The card title attribute specifies the title that appears on the card
+   */
+  @Prop({ reflect: true }) type: 'link' | 'action' = 'link';
+
+  /**
+   * The card title attribute specifies the title that appears on the card
+   */
+  @Prop({ reflect: true }) cardTitle!: string;
+
+  /**
+   * The card title attribute specifies the title that appears on the card
+   */
+  @Prop() titleElement: 'h3' | 'h4' | 'h5' | 'h6' | 'a' = 'a';
+
+  /**
+   * The href attribute specifies the URL of the page the link goes to
+   */
+  @Prop({ reflect: true }) href!: string;
+
+  /**
+   * The description attribute specifies the body of text that appears on the card
+   */
+  @Prop({ reflect: true }) description: string;
+
+  /**
+   * The tag attribute specifies the tag text that appears above the card title
+   */
+  @Prop({ reflect: true }) tag: string;
+
+  /**
+   * The img src attribute specifies the path to the image
+   */
+  @Prop({ reflect: true }) imgSrc: string;
+
+  /**
+   * The img alt attribute specifies the alt text for the image provided, if none, image will be decorative
```

Comment:
Should we add "and will not be read out to screen reader users" or something like that?

Reply:
I don't think that will be necessary, saying the image will be decorative should cover that assistive technology won't pick it up.